packages/components-angular/tests/e2e/specs/tabs-bar-navigation-bug.e2e.ts

```diff
@@ -0,0 +1,31 @@
+import type { Page } from 'puppeteer';
+import { goto } from '../helpers';
+
+let page: Page;
+beforeEach(async () => (page = await browser.newPage()));
+afterEach(async () => await page.close());
+
+it('should correctly set activeTabIndex on p-tabs-bar', async () => {
+  await goto(page, 'tabs-bar-navigation-bug');
+
+  const getTabsBarActiveTabIndex = (): Promise<number> => page.$eval('p-tabs-bar', (el) => (el as any).activeTabIndex);
+  const getHeadingText = (): Promise<string> => page.$eval('p-heading', (el) => el.textContent);
+
+  expect(await getTabsBarActiveTabIndex()).toBe(0);
+  expect(await getHeadingText()).toBe('Bug Page 1');
```

Comment:
Could also check if tabs-bar underline style was applied correctly

Reply:
That should be covered by components-js e2e tests already. So we don't get any additional value.

Reply:
Wasn't this broken only on angular before?

Reply:
Broken was that the `activeTabIndex` was reset to undefined. As a visual result the line was gone but that's more like the symptom.

Reply:
Ok i thought there was a case where only the style was missing but correct `activeTabIndex`, if that never was the case we should be safe